fhir-mongoid-models.gemspec

```diff
@@ -27,17 +27,17 @@ Gem::Specification.new do |spec|
 
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
-  spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
+  spec.files = Dir.chdir(File.expand_path('..', __FILE__)) do
```

Comment:
Codacy found an issue: [Use `expand_path(__dir__)` instead of `expand_path('..', __FILE__)`.](https://app.codacy.com/gh/MeasureAuthoringTool/fhir-mongoid-models/pullRequest?prid=5760802)